pkgs/applications/networking/cluster/argocd-autopilot/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "argocd-autopilot";
+  version = "0.3.0";
+  tag = "v${version}";
+  commit = "c8d17bef976649e4dc2428c14c39e30a0f846552";
+
+  src = fetchFromGitHub {
+    owner = "argoproj-labs";
+    repo = "argocd-autopilot";
+    rev = "${tag}";
```

Comment:
```suggestion

  src = fetchFromGitHub {
    owner = "argoproj-labs";
    repo = "argocd-autopilot";
    rev = "v${version}";;
```

Reply:
This removes the commit and adds an extra semi-colon at the end of rev.

I'm going to make these changes manually, but @bryanasdev000 asked me to add back the commit. So what should it be?

Reply:
> This removes the commit and adds an extra semi-colon at the end of rev.
> 
> I'm going to make these changes manually, but @bryanasdev000 asked me to add back the commit. So what should it be?

I think by default, a lot of packages Go goes without it, but based on @thiagokokada feedback on https://github.com/NixOS/nixpkgs/pull/163522 we need it to make sure that the application does not behave out of ordinary.

Also, to bot update it will need to work on a update script or else we will need to update it by hand each time.

Reply:
I'm fine with either, just tell me which one should I leave in this PR (we can always add back the commit later though).

Reply:
> I'm fine with either, just tell me which one should I leave in this PR (we can always add back the commit later though).

I am with you, but the call is basically from who merges it.

In the last update of ArgoCD I dropped it, but on @thiagokokada  comments I added it.

In the end, a updateScript will be needed for this package as long as we have different hashes, so may vote for now will probably rest on maintain this with commit and maintain some "default" for argo-* expressions.